src/components/view/DetailsTab.vue

```diff
@@ -23,7 +23,8 @@
       <div>
         <strong>{{ $t(item) }}</strong>
         <br/>
-        <div>
+        <div v-if="resource[item]==='Unlimited'">-1</div>
```

Comment:
@svenvogel @PaulAngus @borisstoyanov  do we want to show -1 instead of unlimited?

Reply:
correct, but I revised the display data and removed it as it is now not necessary